utils/extract-headers.sh

```diff
@@ -1,61 +1,89 @@
-#!/bin/sh
-
-ANDROID_ROOT=$1
-HEADERPATH=$2
-MAJOR=$3
-MINOR=$4
-
-PATCH=$5
-PATCH2=$6
-PATCH3=$7
+#! /bin/sh
```

Comment:
A space before `/bin/sh` ? :open_mouth:

Reply:
Not to butt in here, but it does help when `grep`ping for `#!/bin/sh`.

Reply:
I just copy & pasted from Mer. No code from me ;) Happy to change it either way.

Reply:
Well spaces in shebang is actually completely ok; see also https://unix.stackexchange.com/questions/276751/is-space-allowed-between-and-bin-bash-in-shebang for some more details.
As it would let us have an identical version as Mer's, it could be a good point.

Reply:
should we fix this in the mer version then? also is there any better way how we can keep those two files in sync?